cognite/powerops/client/powerops_client.py

```diff
@@ -43,7 +43,7 @@ def __init__(
         self._write_dataset = write_dataset or settings.powerops.write_dataset
         self._cogshop_version = cogshop_version or settings.powerops.cogshop_version
 
-        self.cdf = get_cognite_client(config)
+        self.cdf = CogniteClient(config=config) if config else get_cognite_client()
```

Comment:
I think the `get_cognite_client()` is a bit magical. I suggest adding a factory method to explicitly get the client from settings.
Something like:
```python
@classmethod
def create_from_settings(cls, read_dataset, write_dataset, version):
     config = get_client_config_from_settings()
    return(read_dataset, write_dataset, version, config)
```